FinSearchUnified/Bundle/ProductNumberSearch.php

```diff
@@ -163,13 +174,28 @@ private function getFacetHandler(SimpleXMLElement $filter)
 
     /**
      * @param Criteria $criteria
+     * @param ShopContextInterface $context
      * @param SimpleXMLElement|null $filters
      *
      * @return array
+     * @throws Zend_Cache_Exception
      */
-    protected function createFacets(Criteria $criteria, SimpleXMLElement $filters = null)
+    protected function createFacets(Criteria $criteria, ShopContextInterface $context, SimpleXMLElement $filters = null)
     {
         $facets = [];
+        $url = md5(Shopware()->Front()->Request()->getRequestUri());
+        $cacheId = sprintf('finsearch_%s', $url);
+
+        if ($this->cache->load($cacheId) === false) {
+            /** @var QueryBuilder $query */
+            $query = $this->queryBuilderFactory->createSearchNavigationQuery($criteria, $context);
+            $response = $query->execute();
+            $this->cache->save($response, $cacheId, ['FINDOLOGIC'], 86400);
```

Comment:
Nobody knows what `86400` really is. It is much easier by just putting a formula like `60 * 60 * 24`, because everyone knows that this is a day, by just looking at it.